src/Query/QueryBuilder.php

```diff
@@ -515,7 +515,7 @@ public function forUpdate(ConflictResolutionMode $conflictResolutionMode = Confl
      *
      * <code>
      *     $qb = $conn->createQueryBuilder()
-     *         ->union('SELECT 1 AS field1', 'SELECT 2 AS field1');
+     *         ->union('SELECT 1 AS field1');
```

Comment:
There is no point in having just one call to `union`, and no call to `addUnion`, is there? In fact, does that even produce a valid query? I think you should use the same example in both cases.

Reply:
Fixed!